dotnet/SimpleSauce/SauceSession.cs

```diff
@@ -9,26 +10,38 @@ public class SauceSession
     {
         public ChromeOptions ChromeOptions { get; private set; }
         public DataCenter DataCenter { get; set; } = DataCenter.UsWest;
+        public SauceOptions Options => _options;
 
-        private readonly IRemoteDriver _remoteDriverManager;
-        private readonly SauceOptions options;
+        public readonly IRemoteDriver _driverImplementation;
+        private readonly SauceOptions _options;
 
         public SauceSession()
         {
-            _remoteDriverManager = new ConcreteRemoteWebDriver();
+            _driverImplementation = new ConcreteRemoteWebDriver();
+            _options = new SauceOptions();
         }
-        public SauceSession(IRemoteDriver driverManager)
+        public SauceSession(IRemoteDriver driver)
         {
-            _remoteDriverManager = driverManager;
+            _driverImplementation = driver;
+            _options = new SauceOptions();
         }
 
         public SauceSession(SauceOptions options)
         {
-            this.options = options;
+            this._options = options;
+            _driverImplementation = new ConcreteRemoteWebDriver();
+        }
+
+        public SauceSession(SauceOptions options, IRemoteDriver driver)
+        {
+            this._options = options;
+            _driverImplementation = driver;
         }
 
         public IWebDriver Start()
         {
+            if (_options.EdgeOptions != null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4510198)